AmplifyPlugins/Logging/Sources/AWSCloudWatchLoggingPlugin/Configuration/DefaultRemoteLoggingConstraintsProvider.swift

```diff
@@ -58,7 +58,9 @@ public class DefaultRemoteLoggingConstraintsProvider: RemoteLoggingConstraintsPr
         let loggingConstraint = try JSONDecoder().decode(LoggingConstraints.self, from: data)
         loggingConstraintsLocalStore.setLocalLoggingConstraints(loggingConstraints: loggingConstraint)
 
-        if let etag = (response as? HTTPURLResponse)?.value(forHTTPHeaderField: "If-None-Match") {
+        if let etag = (response as? HTTPURLResponse)?.value(forHTTPHeaderField: "ETag") {
+            loggingConstraintsLocalStore.setLocalLoggingConstraintsEtag(etag: etag)
+        } else if let etag = (response as? HTTPURLResponse)?.value(forHTTPHeaderField: "If-None-Match") {
```

Comment:
i think it's worth adding a comment in code to explain why the etag may be in the `If-None-Match` header, is it because we provided some lambda code to the developer and they may have deployed it already so we want to be backwards compatible? How did we decide to use ETag instead?

Reply:
Good suggestion, I've updated with code comments.  ETag should be use based on the Http header spec and is consistent with the updated lambda sample code.